tests/br_300_small_tables/run.sh

```diff
@@ -29,7 +32,22 @@ done
 
 # backup db
 echo "backup start..."
+
+export GO_FAILPOINTS="github.com/pingcap/br/pkg/task/progress-call-back=return(\"$PROGRESS_FILE\")"
 run_br backup db --db "$DB" -s "local://$TEST_DIR/$DB" --pd $PD_ADDR
+export GO_FAILPOINTS=""
+
+# check if we have 300 range, because we have 300 tables
+if [[ "$(wc -l <$PROGRESS_FILE)" == "1" ]] && [[ $(grep -q "range:300" $PROGRESS_FILE) ]];
```

Comment:
Seems the return code of `grep` is enough. And seems each table has an index...?

```suggestion
if [[ "$(wc -l <$PROGRESS_FILE)" == "1" ]] && grep -q "range:600" $PROGRESS_FILE;
```

Reply:
ok, I removed the number ...

Consider PKIsHandle in release-4.0, if we compare the number, it will make the case more complicated...

Reply:
Yep, and seems clustered indices also influence this...